src/blockserver/server.py

```diff
@@ -105,7 +106,7 @@ def initialize(self, transfer_cls, get_auth_cls, get_cache_cls, database_pool,
         self.streamer = None
         await self._authorize_request()
         if self.request.method == 'POST':
-            self.request.connection.set_max_body_size(options.max_body_size)
+            self.remaining_upload_size = options.max_body_size
```

Comment:
I think max_body_size is still needed because the connection itself enforces it.